src/main/java/se/techinsight/urlshortener/UrlController.java

```diff
@@ -21,47 +20,47 @@
 @Tag(name = "Url Controller", description = "API for working with URL Controller")
 @RestController
 @Validated
-@RequestMapping({"/api/url"})
+@RequestMapping(value = {"/api/url"})
 public class UrlController {
 
     @Autowired
     private UrlShortenerService service;
 
     @Operation(
             tags = "Development",
-            summary = "Get URL by ID",
+            summary = "Get URL object by {id}",
             description = "Developers endpoint, for debug reasons only")
     @GetMapping(value = "/id/{id}", produces = MediaType.APPLICATION_JSON_VALUE)
     public UrlShortener get(@PathVariable("id") Long id) {
         return service.findById(id);
     }
 
-    @Operation(summary = "Get URL by shorten_key")
+    @Operation(summary = "Get URL object by {shorten_key}")
     @GetMapping(value = "/{shorten_key}", produces = MediaType.APPLICATION_JSON_VALUE)
-    public UrlShortener getByKey(@PathVariable("shorten_key")
-                                 @Size(min = 1, max = 6, message = "must be longer then 1 and less then 6")
-                                         String shortenKey) {
+    public UrlShortener getByKey(@PathVariable("shorten_key") @Size(min = 1, max = 6) String shortenKey) {
         return service.findByShortenKey(shortenKey);
     }
 
+    @Operation(summary = "Create URL object")
     @ResponseStatus(HttpStatus.CREATED)
     @PostMapping(value = "/", consumes = MediaType.APPLICATION_JSON_VALUE, produces = MediaType.APPLICATION_JSON_VALUE)
     public UrlShortener create(@Valid @RequestBody OriginalUrlDto dto) {
         return service.save(dto);
     }
 
+    @Operation(summary = "Delete URL object by {shorten_key}")
+    @ResponseStatus(HttpStatus.ACCEPTED)
     @DeleteMapping(value = "/{shorten_key}", produces = MediaType.APPLICATION_JSON_VALUE)
-    public ResponseEntity delete(@PathVariable("shorten_key") @Size(min = 1, max = 6) String shortenKey) {
+    public Map<String, Object> delete(@PathVariable("shorten_key") @Size(min = 1, max = 6) String shortenKey) {
```

Comment:
Why do you return the map here? Is ACCEPTED status code will be not enough for the front-end to understand that operation finished successfully?

Reply:
probably yes, but i wanted to show FE dev or consumer of this API that operation finished successfully, or it is bad practice, the response status 202 will be enough?